q2cli/builtin/tools.py

```diff
@@ -195,7 +196,104 @@ def peek(path):
         click.echo(metadata.format)
 
 
-@tools.command('inspect-metadata',
+@tools.command(name='cast-metadata',
+               short_help='Designate metadata column types.',
+               help='Designate metadata column types.'
+                    ' Supported column types can be found on docs.qiime2.org.'
+                    ' Providing multiple file paths to this command will merge'
+                    ' the metadata.',
+               cls=ToolCommand)
+@click.option('--cast', required=True, metavar='COLUMN:TYPE', multiple=True,
+              help='Cast parameter for each metadata column that should'
+              ' contain a specified column type (supported types can be'
```

Comment:
"Cast" is a tricksy hobbit in prose, since it can be read as a verb here, which is a bummer. How about something like this:

```
              help='Parameter for each metadata column that should be'
              ' cast as a specified column type (supported types can be'
```